internal/lookoutv2/generate/main.go

```diff
@@ -0,0 +1,156 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/stdcopy"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	armadaMountDir  = "/mnt/armada"
```

Comment:
I think this may be specific for wsl?  Not sure this would work on native linux or mac?